.gitmodules

```diff
@@ -15,10 +15,10 @@
 	path = externals/kokkos
 	url = git@github.com:E3SM-Project/kokkos.git
 	branch = e3sm/kokkos
-[submodule "components/cam/src/physics/rrtmgp/external"]
```

Comment:
Was this not already fixed in the main scream branch?  How are we able to run F-Compsets if we are still pulling in a "cam" directory?